event_firstname/models/event_registration.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from odoo import models, fields, api
+
+
+_logger = logging.getLogger(__name__)
+
+
+class EventRegistration(models.Model):
+
+    _inherit = "event.registration"
+
+    firstname = fields.Char(string="Firstname",
```

Comment:
mixing single quotes w/ double quotes
also, put a newline right after the first parenthesis :)